### PR TITLE
hw 01 done

### PR DESCRIPTION
**Pull request by longxuan0908**

### CODE REVIEW

**Review by sullivannicole — COMMENTED** [review]
Coding style: NA
Coding strategy: NA
Achievement, creativity: +
Ease of access: +

Hi Xuan!

Great, concise autobio!  You used four elements effectively:  heading (via pound sign), italics (via underscore), list (via numerics and period), and a link!

My only suggestion is:  in future, include your methods and difficulties as a comment on the pull request, not within the actual body of your repo/html.  I was unsure about where to include this as well, but when I checked the instructions, it said to convey this information through a comment, and other students used this method as well.

Overall, met the expectations!  Kudos!

**Comment by yuepan4** [comment]
Hi Xuan,

I really like your autobiography! Apparently you managed the skills required by this assignment. I had the same feeling when I was working on this assignment: easier to code than figure our how to push and pull between local Rstudio and Github. 
Overall it definitely met the requirement of this assignment, perhaps the only piece of advice I can think of is maybe to try out inserting pictures when you play around with Rstudio privately.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+2. Analysis of Micro Data
+3. This Class
+
+You can learn more about the MAPSS program through its website:
+https://mapss.uchicago.edu/
+
+# Report
+I created a new project from version control and pulled from github using git pull command. Then I modified the markdown file in Rstudio. 
+
+The hard problem is to find out the R Markdown code seems to require blank line before the list in order for list syntax to work properly. Trying to understand how commit and push work is also the harder part. The other parts are relatively easy.
```
Comment (Alex-A14) [inline README.md:21]: I also had issues with creating lists and it seems like others did too.  Just like you I found that using a blank line in between helps.
Based on my time reviewing other's work it seems that commits will save versions of your work to GitHub for review if needed.

**Review by Alex-A14 — COMMENTED** [review]
Clean and clear work with the appropriate concepts used.

Good work.
Best,
Your fellow MAPSS student.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,21 @@
+---
+output: html_document
```
Comment (bensoltoff) [inline README.md:2]: Note that this is unnecessary for Markdown documents we intend to view on GitHub. GitHub does all the rendering for us automatically, so there is no need to generate the HTML document. You can generate a preview of this within R Studio, but still don't need to add the `output: html_document` command

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
...
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
+2. Analysis of Micro Data
+3. This Class
+
+You can learn more about the MAPSS program through its website:
+https://mapss.uchicago.edu/
```
Comment (bensoltoff) [inline README.md:16]: Alternatively, you could write this `You can learn more about the MAPSS program through [its website](https://mapss.uchicago.edu/)`, which when rendered looks like

You can learn more about the MAPSS program through [its website](https://mapss.uchicago.edu/)

File: README.md (hunk 1 above)
Comment (bensoltoff) [reply README.md:21]: Yup. It's a bit of a quirk in Markdown, but that's just how it works. Also note that in the future this feedback should be included in the summary of the pull request, not in the actual assignment repo

File: README.md [hunk 4]
```diff
@@ -1,2 +1,21 @@
...
+output: html_document
+---
 # hw01
 Repository for first CFSS homework assignment
+
+# Introduction
+I am a graduate student in the _MAPSS_ program with an economics concentration. 
+The list of classes that I am taking this quarter include:
+
+1. Firm Dynamics and Innovation
```
Comment (bensoltoff) [inline README.md:11]: Note that you don't have to number the items yourself. What I mean is this:

```
1. Firm Dynamics and Innovation
1. Analysis of Micro Data
1. This Class
```

Renders like this:

1. Firm Dynamics and Innovation
1. Analysis of Micro Data
1. This Class

**Review by bensoltoff — COMMENTED** [review]
Evaluation HW-01 for Longxuan Wang
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       x       |            |
| **Coding strategy**         |           |       x       |            |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Overall a solid first submission
* You got the basic syntax for formatting text. Now focus on how to add links and images to Markdown documents. They use a bit more complex formatting but will be incredibly useful in your later assignments
* Make sure to commit frequently. You don't show any history or evolution of your repo. Commit whenever you finish writing or modifying code for a specific task or chunk of code. It doesn't mean commit after every line of code - try to find a balance that allows you to review your code history.

**Pull request closed without merging**